service/selectors/entities/users.js

```diff
@@ -61,10 +60,9 @@ export const getProfilesInCurrentChannel = createSelector(
             });
         });
 
-        // We could get rid of this if server side sorting is a possibility
         const sortedCurrentProfiles = currentProfiles.sort((a, b) => {
-            const nameA = displayUsername(a, preferences);
-            const nameB = displayUsername(b, preferences);
+            const nameA = a.username;
+            const nameB = b.username;
```

Comment:
now that I'm looking at this again i'm wondering if we should consider using `localeCompare` here to handle non-english chars